app/models/suggestion.py

```diff
@@ -0,0 +1,119 @@
+from .. import db
+from datetime import datetime
+import pytz
```

Comment:
These 2 imports should go before `from .. import db` since they are python packages.